bundles/org.openhab.binding.noolite/src/main/java/org/openhab/binding/noolite/handler/NooliteMTRF64BridgeHandler.java

```diff
@@ -0,0 +1,206 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.noolite.handler;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.smarthome.core.thing.Bridge;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.binding.BaseBridgeHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.openhab.binding.noolite.NooLiteBindingConstants;
+import org.openhab.binding.noolite.internal.NooliteMTRF64Adapter;
+import org.openhab.binding.noolite.internal.config.NooliteBridgeConfiguration;
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ *
+ * @author Petr Shatsillo - Initial contribution
+ *
+ */
+@NonNullByDefault
+public class NooliteMTRF64BridgeHandler extends BaseBridgeHandler {
+
+    private static Logger logger = LoggerFactory.getLogger(NooliteMTRF64BridgeHandler.class);
+    @Nullable
+    static NooliteMTRF64Adapter adapter;
```

Comment:
Adapter should not be static and be private and probably can be initialized here.